include/Application.hpp

```diff
@@ -15,10 +17,13 @@ namespace ccm {
 		~Application();
 
 		void render();
+		void handleEvents();
+
 		bool m_quit{ false };
 
 	private:
 		SDL_Window* m_window{ nullptr };
 		SDL_Renderer* m_renderer{ nullptr };
+		SDL_Event m_event;
```

Comment:
Do we really need the event to be part of the Application class?

Reply:
Shouldn't the Application class handle everything it needs? When the project is bigger, we wouldn't want to have the event loop in main as we'd have to be changing Application state, right?

Reply:
Yes but what itay means is that the SDL_Event variable can be local to the `handleEvents()` function

Reply:
I changed it